packages/carbon-web-components/src/components/tile/tile-story.ts

```diff
@@ -184,6 +281,105 @@ export const expandable = (args) => {
   `;
 };
 
+export const ExpandableWithInteractive = (args) => {
+  const { colorScheme, expanded, disableChange, onBeforeChange, onChange } =
+    args?.[`${prefix}-expandable-tile`] ?? {};
+  const handleBeforeChanged = (event: CustomEvent) => {
+    onBeforeChange(event);
+    if (disableChange) {
+      event.preventDefault();
+    }
+  };
+  return html`
+    <cds-expandable-tile
```

Comment:
weird interaction happening when i click right on the caret button
![Mar-20-2023 13-45-57](https://user-images.githubusercontent.com/20210594/226449407-c17f30c0-9a26-4883-a0aa-d1be19b0449e.gif)